conda_content_trust/signing.py

```diff
@@ -74,7 +73,7 @@ def wrap_as_signable(obj):
     Raises ❌TypeError if the given object is not a JSON-serializable type per
     SUPPORTED_SERIALIZABLE_TYPES
     """
-    if not type(obj) in SUPPORTED_SERIALIZABLE_TYPES:
+    if not isinstance(obj, SUPPORTED_SERIALIZABLE_TYPES):
```

Comment:
I would want to show this error after calling json.dumps() not as a pre-check.

Reply:
yeah I've looked at this before, we don't use `isinstance` here because we don't want to support subclasses

Reply:
Changed back. Should switch to a dumps/loads or restructure the way this function works - this check is only testing the outer level container anyway so a list of object() would pass the check, but fail json.dumps()